appdatabase/migrations/sql/1742299652_add_news_notifications_setting.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings ADD COLUMN news_notifications_enabled BOOLEAN DEFAULT FALSE;
```

Comment:
I wonder if we could make this an enum (combobox) where the options could be `Enabled with push notification`, `Enabled with no notification`, `Disabled`

I can see a world where someone is interested by News, but doesn't want push notifications for them.
I guess on Desktop it would be `OS notifications` instead.

cc @xAlisher @benjthayer

Reply:
For future readers, we as a group discussed the desired values for this setting. What we've decided is that we would have two settings:
  * One for controlling when news notifications are allowed via `news_notifications_enabled` setting
  * And another for controlling when the news feed feature is active via `news_feed_enabled` setting